### DISCUSSION TITLE
MON users not working after Upgrade to 0.25

### DISCUSSION THREAD

**Discussion by kleuf** [tchiotludo/akhq#1862] [Q&A]
Hi there

I upgraded from 0.23 to 0.25 and since then, the usual kafkahqmon user and other mon users does not work anymore.
Where can i start looking at this and resolve ?

**Comment by AlexisSouquiere** [comment]
Start from here: https://akhq.io/docs/configuration/authentifications/groups.html and the warning explaining the breaking changes in RBAC in 0.25.0+
